app/Http/Controllers/Front/ClientController.php

```diff
@@ -286,11 +286,14 @@ public function getOrders()
                             ->addColumn('product_name', function ($model) {
                                 return $model->product()->first()->name;
                             })
+                            ->addColumn('number', function ($model) {
+                              return '<a href='.url('my-order/'.$model->id).'>'.$model->number.'</a>';
+                            })
+                            ->addColumn('version', function ($model) {
+                              return getVersionAndLabel($model->subscription()->first()->version,$model->product,'badge');
```

Comment:
do not add any query in an ongoing look. Please use joins and append version beforehand

Reply:
fixed